mipengine/node/monetdb_interface/monet_db_facade.py

```diff
@@ -125,6 +128,11 @@ def error_handling(**kwargs):
             try:
                 return func(**kwargs)
             except Exception as exc:
+                if isinstance(
+                    exc, ProgrammingError
+                ) and "is already part of MERGE TABLE" in str(exc):
+                    return
+
```

Comment:
I am not sure what this is doing here. Is this needed? If it is, why don't you return anything?